syntax = "proto3";
message RespondMsg
{
	int32 status = 1;
	int32 secKeyID = 2;
	bytes clientID = 3;
	bytes serverID = 4;
	string data = 5;
}
